src/ui/osx/TogglDesktop/Feature/Timeline/TimelineActivityRecorderViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TimelineActivityRecorderViewController.swift
+//  TogglDesktop
+//
+//  Created by Nghia Tran on 1/3/20.
+//  Copyright © 2020 Alari. All rights reserved.
+//
+
+import Cocoa
+
+protocol TimelineActivityRecorderViewControllerDelegate: class {
+
+    func timelineActivityRecorderShouldDidClickOnCloseBtn(_ sender: Any)
+}
+
+final class TimelineActivityRecorderViewController: NSViewController {
+
+    private struct Constants {
+        static let LearnMoreURL = "https://toggl.com"
```

Comment:
@IndrekV when we have new URL for the blog, please let me know to update it 👍

Reply:
We will not have it before this release. This new article needs to be compiled when the release is out so that support people writing the article can test and make screenshots. Right now just please remove the "learn more" part from the design.

Reply:
All done. I hide it instead of removing completely as well as adjust the padding to make the content fits 👍